pypodio2/api.py

```diff
@@ -28,10 +28,12 @@ def OAuthAppClient(client_id, client_secret, app_id, app_token, user_agent=None,
     auth = transport.OAuthAppAuthorization(app_id, app_token,
                                            client_id, client_secret, domain)
 
-    return AuthorizingClient(domain, auth, user_agent=user_agent)
+    return AuthorizingClient(domain, auth, user_agent=user_agent, app_id=app_id)
 
 
-def AuthorizingClient(domain, auth, user_agent=None):
+def AuthorizingClient(domain, auth, user_agent=None, app_id=None):
     """Creates a Podio client using an auth object."""
     http_transport = transport.HttpTransport(domain, build_headers(auth, user_agent))
-    return client.Client(http_transport)
+    c = client.Client(http_transport)
```

Comment:
Can we rename this variable to `client_` or something similar? In general we would prefer to avoid one letter variables as it might interfere with some tools,  e.g. `c` is a command in [`pdb`](https://docs.python.org/3/library/pdb.html#pdbcommand-continue).

Reply:
100%.
```suggestion
    client_ = client.Client(http_transport)
```